packages/flutter/lib/src/cupertino/scrollbar.dart

```diff
@@ -125,6 +126,54 @@ class CupertinoScrollbar extends StatefulWidget {
   /// {@endtemplate}
   final ScrollController controller;
 
+  /// {@template flutter.cupertino.cupertinoScrollbar.displayAlways}
+  /// A [bool] property for showing [Scrollbar] always.
+  ///
+  /// To show [ScrollBar] always you need to make this property true,
+  /// however the [controller] shouldn't be null.
+  ///
+  /// You should pass the [Scrollable]'s [ScrollController] to [controller].
+  ///
```

Comment:
Some revision on the docs above:

```
Indicates whether the [Scrollbar] should always be visible.

When false, the scrollbar will be shown during scrolling and will fade out otherwise.

When true, the scrollbar will always be visible and never fade out.  The [controller] property must be set in this case.  It should be passed the relevant [Scrollable]'s [ScrollController].

Defaults to false.
```